distributed/http/scheduler/tests/test_scheduler_http.py

```diff
@@ -259,6 +259,15 @@ async def test_api(c, s, a, b):
             assert (await resp.text()) == "API V1"
 
 
+@gen_cluster(client=True, clean_kwargs={"threads": False}, security=tls_only_security())
+async def test_api_disabled_if_secure(c, s, a, b):
+    async with aiohttp.ClientSession() as session:
+        async with session.get(
+            "http://localhost:%d/api/v1" % s.http_server.port
+        ) as resp:
+            assert resp.status == 404
```

Comment:
if there is a typo, the route is actually removed or the version number is bumped, this test would still pass.
You could parametrize over security and ensure that the route is reachable if security is disabled to make sure the test assumption is actually valid